src/io/image_aug_default.cc

```diff
@@ -277,7 +413,8 @@ class DefaultImageAugmenter : public ImageAugmenter {
     }
 
     // crop logic
-    if (param_.max_crop_size != -1 || param_.min_crop_size != -1) {
+    if (!param_.random_resized_crop &&
```

Comment:
This looks weird. Why not resize in the random sized crop section?

Reply:
random resized crop considers sample the relative area and the aspect ratio of the to-be-cropped part.

here's random crop, it randomly crop a square with its edge length sampled from [min_crop_size, max_crop_size]